src/styles.scss

```diff
@@ -1,6 +1,6 @@
-/* You can add global styles to this file, and also import other style files */
 @import "~bootstrap/dist/css/bootstrap.css";
 @import "./scss/mixins";
 @import "./scss/variables";
 @import "./scss/fonts";
 @import "./scss/reset";
+@import "../src/app/shared/components/loader/loader.component.scss";
```

Comment:
remove this.

use ViewEncapsulation.None for loader-component to apply .spinner styles globaly